test/db/SQLJSPackageDB.test.ts

```diff
@@ -77,6 +76,11 @@ describe('SQLJSPackageDB', () => {
       packageDb.saveResourceInfo(valueSetFour);
     });
 
+    it('should not throw even if the db is not initialized', () => {
+      const unitializedPackageDB = new SQLJSPackageDB();
+      unitializedPackageDB.clear();
```

Comment:
I only say this because I think it would bug you too, but this should be `uninitializedPackageDB`
```suggestion
      const uninitializedPackageDB = new SQLJSPackageDB();
      uninitializedPackageDB.clear();
```

And sadly, that's the case for the other places in this file. But I will gladly add suggestions for each of those if it's helpful!

Reply:
Ugh. My fingers love typing that word out wrong and usually I catch it. But... I missed it this time and then did a lot of copy/paste!

Reply:
Fixed in 10e109d.